app/src/test/java/com/mstudio/superheroarch/localdatasource/mapper/CharacterLocalEntityMapperTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.mstudio.superheroarch.localdatasource.mapper
+
+import com.mstudio.superheroarch.RickAndMortyRepositoryInstruments
+import com.mstudio.superheroarch.localdatasource.model.toCharactersRemoteEntity
+import org.junit.Assert
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class CharacterLocalEntityMapperTest {
+
+    @Test
+    fun `given a character local entity, when mapped to character remote entity, the has correct values`() {
```

Comment:
Are mapper test really useful? think if it makes sense to have specific test for them or could be included in other tests.